TestProjects/UniversalGraphicsTest_Lighting/Assets/Test/Editor/ReflectionProbeTests.cs

```diff
@@ -0,0 +1,149 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using NUnit.Framework;
+using UnityEngine.TestTools.Graphics;
+using UnityEditor.SceneManagement;
+using UnityEditor;
+
+public class Editmode_ParametricReflectionProbeTests
+{
+	private readonly string sceneOutputPath = "Assets/EditModeTestAssets/Lighting_ReflectionProbeBaking/ReflectionProbeBake";
+	private readonly string sceneFileName = "Assets/EditModeTestAssets/Lighting_ReflectionProbeBaking/ReflectionProbeBake.unity";
```

Comment:
Looks like you don't have the formatting githook installed? https://github.com/Unity-Technologies/Graphics/tree/master/Tools

Reply:
Will install them now. Looks like the installation doesn't work with the new python versions. Will check with @theo-pnv .